### PR TITLE
Refactor naming of getters and setters

### PR DESCRIPTION
**Pull request by cdoehn**
## Description

LaserScan uses getX and setX naming while most of the other classes just use X we should unify this naming. The old getters and setters are now deprecated.

### Things to add, update or check by the maintainers before this PR can be merged.

- [x] Public api function documentation
- [x] Architecture documentation reflects actual code structure
- [x] ~Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)~
- [x] ~Overview on [ROS wiki](https://wiki.ros.org/pilz_robots)~
- [x] ~Package Readme ([example pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md))~
- [x] ~Good commit messages ([some tips](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9))~
- [x] ~CHANGELOG.rst updated~
- [x] ~Copyright headers~
- [x] ~Examples~

### Review Checklist
- [x] Soft- and hardware architecture (diagrams and description)
- [x] Test review (test plan and individual test cases)
- [x] ~Documentation describes purpose of file(s) and responsibilities~
- [x] Code (coding rules, style guide)

### Release planning (please answer)
- [x] ~When is the new feature released?~
- [x] ~Which dependent packages do have to be released simultaneously?~

### Hardware tests
_Unstrike the text below to enable automatic hardware tests if available. Otherwise use this as a request for the reviewer._
- [x] ~Perform hardware tests~

</details>

### CODE REVIEW

File: src/active_zoneset_node.cpp [hunk 1]
```diff
@@ -77,6 +77,11 @@ bool ActiveZonesetNode::isAllInformationAvailable() const
   return active_zoneset_id_.is_initialized() && zoneset_config_.is_initialized();
 }
 
+ZoneSet ActiveZonesetNode::activeZoneset() const
```
Comment (agutenkunst) [inline src/active_zoneset_node.cpp:80]: I think this should be called by the deprecated function.

Comment (agutenkunst) [reply src/active_zoneset_node.cpp:80]: Holds for all future changes. This should also resolve your coverage issues.

Comment (cdoehn) [reply src/active_zoneset_node.cpp:80]: > This should also resolve your coverage issues.
No it doesn't solve the coverage issues, but I will do it anyway

**Comment by rfeistenauer** [comment]
Thanks so far. Two things:
1. this needs a rebase to solve the conflicts.
2. do we really need to deprecate the getters of our internal classes? I would suggest doing it only for the public api.

**Review by agutenkunst — APPROVED** [review]
Thank you. Feel free to merge.

**Pull request merged**